pkg/docker/docker.go

```diff
@@ -178,6 +182,16 @@ func (c *Client) DownloadImage(imageSource, targetFile string) (v1.Image, error)
 	return img, nil
 }
 
+// GetRemoteImageInfo retrieves information about the image (e.g. digest) without actually downoading it
+func (c *Client) GetRemoteImageInfo(imageSource string) (v1.Image, error) {
+	ref, err := name.ParseReference(imageSource, name.WeakValidation)
```

Comment:
```suggestion
	ref, err := c.getImageRef(imageSource)
```

Reply:
`getImageRef` respects if a default registry is set, your implementation doesn't right at the moment.